src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -181,4 +184,20 @@ public function exists(Key $key)
 
         return false;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsPutOffExpiration(): bool
+    {
+        return true;
```

Comment:
check if at least 1 store supports expiration.

At line  118 you should also check if the stores supports expiration and throws an exception if none of theme support it